### DISCUSSION TITLE
Does the `conditions` key work inside of a `schema` object?

### DISCUSSION THREAD

**Discussion by fablexis** [vueform/vueform#169] [Questions] [answered]
I've read the docs about the conditional rendering, but I realized that the `conditions` key when having it inside of a `schema` key, does not work... 

take a look at this example:

```
            relationship: {
              type: 'object',
              schema: {
                divider: {
                  type: 'static',
                  tag: 'hr'
                },
                p: {
                  type: 'static',
                  content: 'Emergency Contact Relationship to Child',
                  description: 'Do you want to enable a list for this in the form?',
                  columns: {
                    container: 10,
                    label: 12,
                    wrapper: 12
                  },
                  tag: 'p'
                },
                emergencyRelationToggle: {
                  type: 'toggle',
                  columns: {
                    container: 2,
                    label: 12,
                    wrapper: 12
                  },
                  align: 'right',
                  default: true
                },
                emergencyContactRelation: {
                  type: 'text',
                  label: 'Emergency Contact Relationship to Child',
                  conditions: [['emergencyRelationToggle', true]]
                }
              }
            }
           ```
           
           The `conditions` for `emergencyRelationToggle` does not work, but if I took `emergencyContactRelation` and `emergencyRelationToggle` out of the `schema`, it works... 
           
           Am I missing something here?
           
           @adamberecz

**Comment by fablexis** [comment, marked as answer]
For those struggling with this, the correct way to reference a value nested inside a `schema` object is by accessing its container key name followed by the key name of the element. So, for instance, based on the example of my previous comment, the parent key is `relationship`, since I'm trying to reference the `emergencyRelationToggle` key the way I should point to that value in my `conditions` key, should be:

`conditions: [['relationship.emergencyRelationToggle', true]]`

**Reply by adamberecz** [reply]
Yes, that's the catch!
